code/go/internal/validator/semantic/validate_version_integrity.go

```diff
@@ -130,3 +136,24 @@ func ensureManifestVersionHasChangelogEntry(manifestVersion string, versions []s
 	}
 	return errors.New("current manifest version doesn't have changelog entry")
 }
+
+func ensureChangelogLatestVersionIsGreaterThanOthers(versions []string) error {
+	latestVersion, err := semver.NewVersion(versions[0])
```

Comment:
Just in case, we should check if there is any version in `versions`.

Reply:
:+1: Applied change here: https://github.com/elastic/package-spec/pull/808/commits/02800e4f536cce2134c0f8a17d15bae3ef20ca9f